webpack.config.js

```diff
@@ -44,7 +44,7 @@ if (production) {
 module.exports = [
   {
     context: __dirname,
-    devtool: production ? undefined : 'eval-source-map',
+    devtool: production ? undefined : 'eval',
```

Comment:
>Actually, it is a bit weird that just having a source map makes execution of the code slow. Any theories on why that's causing this?

Vanilla Firefox is apparently not great at handling a 7mb JS file. Tested in Chrome and it loads fine, tested in FirefoxDeveloperEdition and it loads fine, but in normal FF it hangs and sometimes crashes the whole browser.

I'll switch to `cheap-module-eval-source-map` and just use FFDev for testing.